src/ComputeSharp.D2D1/Shaders/Interop/D2D1ShaderInfo.cs

```diff
@@ -53,4 +54,5 @@ public sealed record D2D1ShaderInfo(
     uint MovcInstructionCount,
     uint MovInstructionCount,
     uint InterfaceSlotCount,
-    bool RequiresDoublePrecisionSupport);
\ No newline at end of file
+    bool RequiresDoublePrecisionSupport,
+    D3DFeatureLevel FeatureLevel);
```

Comment:
Should this be named `RequiredFeatureLevel` or `MinFeatureLevel` ? Neither the current name nor the `param` doc comment indicate that it is a requirement

Reply:
Renamed to "MinimumFeatureLevel" to do a C#-y way of keeping the D3D name 🙂